### PR TITLE
DOP-2009: Change Delighted survey options

### PR DESCRIPTION
**Pull request by rayangler**
[[DOP-2009](https://jira.mongodb.org/browse/DOP-2009)]
[[Docs Manual Staging](https://docs-mongodborg-staging.corp.mongodb.com/raymundrodriguez/DOP-2016-1/index.html)]

* Limited the docs properties that Delighted surveys would be shown to.
* Removed `sampleFactors` for Delighted surveys to see if Delighted's [Adaptive Sampling](https://app.delighted.com/docs/api/web#adaptive-sampling) will work.
* Prevent Delighted surveys from showing on staging by checking for the staging url.

### CODE REVIEW

File: themes/mongodb/src/js/componentThirdParty.js
Comment (sophstad) [inline themes/mongodb/src/js/componentThirdParty.js:9]: This is a good use case for a JS [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)!

File: themes/mongodb/src/js/componentThirdParty.js
Comment (sophstad) [inline themes/mongodb/src/js/componentThirdParty.js:1]: The following sites are already off legacy tooling, so perhaps not even worth including them here:
 - `cloud-docs`
 - `datalake`
 - `node`
 - `realm`

**Review by sophstad — CHANGES_REQUESTED** [review]
Nice, just some minor notes!

**Review by sophstad — APPROVED** [review]
One request before merging: let's remove the `package-lock.json` changes from this PR. At this point `docs-tools` is operating on an "if it ain't broke, don't fix it" policy, so would rather not introduce any potential confusion to other users of this repo.

Other than that, LGTM. Thanks Raymund!

File: themes/mongodb/src/js/componentThirdParty.js
Comment (jdestefano-mongo) [inline themes/mongodb/src/js/componentThirdParty.js:1]: What URLs does 'docs' correspond to here?

File: themes/mongodb/src/js/componentThirdParty.js
Comment (jdestefano-mongo) [inline themes/mongodb/src/js/componentThirdParty.js:24]: Does anything here restrict surveys from rendering on our staging environment (https://docs-mongodbcom-staging.corp.mongodb.com/) ?

**Review by jdestefano-mongo — COMMENTED** [review]
@rayangler Thanks for this PR, two questions.

File: themes/mongodb/src/js/componentThirdParty.js
Comment (jdestefano-mongo) [reply themes/mongodb/src/js/componentThirdParty.js:24]: Actually, I just saw your comment in the main PR description. If there is no easy way on legacy, we can punt on that for now and re-evaluate if it becomes a major problem.

Comment (jdestefano-mongo) [reply themes/mongodb/src/js/componentThirdParty.js:24]: Is there a way to retrieve the URL in that part of the code and maybe check for something like the above? Regardless, I'm still good to punt.

File: themes/mongodb/src/js/componentThirdParty.js
Comment (rayangler) [reply themes/mongodb/src/js/componentThirdParty.js:1]: The `docs` repo will be changing its project name from `manual` to `docs` after it is integrated with snooty (so it would map to https://docs.mongodb.com/manual/). I was gonna leave in both for now until everything was all set and changed.

File: themes/mongodb/src/js/componentThirdParty.js
Comment (rayangler) [reply themes/mongodb/src/js/componentThirdParty.js:24]: Yeah, we could probably just check for the staging url in a case like this. I'll make the change and try it out

File: themes/mongodb/src/js/componentThirdParty.js
Comment (rayangler) [inline themes/mongodb/src/js/componentThirdParty.js:16]: @jdestefano-mongo given the [comment](https://github.com/mongodb/docs-tools/pull/484#discussion_r600529348) regarding project names, would we want to honor `Manual` as the project name that's reported on the survey for Delighted?

File: themes/mongodb/src/js/componentThirdParty.js
Comment (jdestefano-mongo) [reply themes/mongodb/src/js/componentThirdParty.js:1]: Can we report https://docs.mongodb.com/manual/ URLs as `manual` in Delighted?

File: themes/mongodb/src/js/componentThirdParty.js
Comment (jdestefano-mongo) [reply themes/mongodb/src/js/componentThirdParty.js:24]: Much appreciated, if it turns out to be a time sink, let me know and we'll punt.

**Review by jdestefano-mongo — APPROVED** [review]
@rayangler LGTM as long as the project name is reported as `manual` to Delighted. I also realize with this change we can no longer test Delighted in staging lol

**Pull request merged**
